lms/views/dashboard.py

```diff
@@ -119,6 +128,9 @@ def _set_lti_user_cookie(self, response):
             value=auth_token,
             secure=not self.request.registry.settings["dev"],
             httponly=True,
+            # Scope the cookie to all the org endpoints
+            # pylint:disable=protected-access
+            path=f"/dashboard/organization/{lti_user.application_instance.organization._public_id}",
```

Comment:
Is `_public_id` always a URL-valid string? Otherwise, we may need to URL-encode it here to make sure it matches the actual URL generated in https://github.com/hypothesis/lms/pull/6213/files#diff-b56c2dd2166bcdd4d14dd9ba083e0265f1522e943a215e4a6d1de87a13b6a815R54 and https://github.com/hypothesis/lms/pull/6213/files#diff-068f7e62b6dad27874177e60c749b03a00101cd998e2975dea672b06b4efb557R29

Reply:
Yup, _public_id is url safe base64: https://github.com/hypothesis/lms/blob/main/lms/models/public_id.py#L45

Reply:
Great